composer.json

```diff
@@ -1,16 +1,25 @@
 {
     "name":        "hostnet/asset-bundle",
     "type":        "symfony-bundle",
-    "description": "Bundle to glue Assets",
+    "description": "Hostnet asset bundle allows for configuring the hostnet/asset-lib from within your symfony app",
     "license":     "MIT",
+    "authors": [
+        {
+            "name":  "Yannick de Lange",
+            "email": "ydelange@hostnet.nl"
+        }
+    ],
     "require": {
-        "php":               ">=7.1",
-        "hostnet/asset-lib": "dev-master",
-        "symfony/symfony":   "^3.3"
+        "php":                      "^7.1",
+        "hostnet/asset-lib":        "dev-master",
```

Comment:
I don't think this dependency is correct

Reply:
For now there is no tag, I will not release this without the lib having a tag but I want to test-run this.